content/code-security/dependabot/working-with-dependabot/guidance-for-the-configuration-of-private-registries-for-dependabot.md

```diff
@@ -39,12 +39,13 @@ You'll find detailed guidance for the setup of the following package managers an
 - [Azure Artifacts](#azure-artifacts)
 - [{% data variables.product.prodname_registry %} registry](#data-variablesproductprodname_registry--registry)
 - [Nexus](#nexus)
+- [ProGet](#proget)
 
 ## Configuring package managers
 
 ### Bundler
 
-Supported by Artifactory, Artifacts, {% data variables.product.prodname_registry %} registry, and Nexus.
+Supported by Artifactory, Artifacts, {% data variables.product.prodname_registry %} registry, Nexus, and ProGet.
```

Comment:
@whatatripp it doesn't seem like ProGet supports Bundler from reading the docs right?

```suggestion
```

Reply:
Yes (Rubygems) - added an example  a bottom :)